src/components/plots/errorBarPlot.ts

```diff
@@ -0,0 +1,47 @@
+module Plottable {
+export module Plot {
+  export class ErrorBar<X> extends AbstractXYPlot<X,number> {
+
+    protected _isVertical: boolean;
+    private _defaultStrokeColor: string;
+
+    /**
+     * Constructs an ErrorBarPlot.
+     *
+     * Error bar plots are used to show variability in a reported measurement on a plot.
+     * They are intended to be merged on top of other sorts of plots.
+     *
+     * @constructor
+     * @param {QuantitativeScale} xScale The x scale to use.
+     * @param {QuantitativeScale} yScale The y scale to use.
+     */
+    constructor(xScale: Scale.AbstractQuantitative<X>, yScale: Scale.AbstractQuantitative<number>, isVertical = true) {
+      super(xScale, yScale);
+      this.classed("error-plot", true);
+      this._defaultStrokeColor = new Scale.Color().range()[1];
```

Comment:
All other plots default to Scale.Color().range()[0]; because this plot is meant to be overlaid on other plots, that means that we won't be able to see parts these error bars by default, which seems questionable to me.